gio/glib-compile-resources.c

```diff
@@ -669,6 +669,12 @@ main (int argc, char **argv)
 	    base[strlen(base) - strlen (".gresource")] = 0;
 	  target_basename = g_strconcat (base, ".c", NULL);
 	}
+      else if (generate_header)
+        {
+       	  if (g_str_has_suffix (base, ".gresource"))
```

Comment:
The coding style does not match the GLib coding style; we use 2 spaces for indentation.

Make sure you're not setting a different value for your Tab key (e.g. using `set ts=2` in vim).